README.md

```diff
@@ -0,0 +1,112 @@
+# Kokkos Documentation Workflow <br/><br/>
+
+Here, we summarize the steps for setting up the kokkos-core-wiki repo, and a
+branch-based workflow for updating documentation in the main Kokkos project.  The kokkos-core-wiki is the
+canonical source of Kokkos documentation "Truth".<br/>
+
+Sandia employees: Creation of wholly new pages will require Review and Approval (R&A).  Check with your Sandia team lead, if you have questions about the R&A process.<br/><br/>
+
+* **Step 1) Create your own fork of the kokkos-core-wiki repo by navigating to `https://github.com/kokkos/kokkos-core-wiki`, and clicking "Fork" in the upper right of the github page**.<br/>
+
+* **Step 2) Clone your forked repo, and add it as a remote:**<br/>
+
+```
+git clone git@github.com:ajpowelsnl/kokkos-core-wiki.git
```

Comment:
This already creates a remote called `origin`. You should `clone` the repository you want to be called `origin`.
In other words, if you want `origin` to point to `kokkos/kokkos-core-wiki` and `ajpowelsnl` to `ajpowelsnl/kokkos-core-wiki` do
```
git clone git@github.com:kokkos/kokkos-core-wiki.git
git remote add ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git
```
In other words, if you want `upstream` to point to `kokkos/kokkos-core-wiki` and `origin` to `ajpowelsnl/kokkos-core-wiki` do
```
git clone git@github.com:ajpowelsnl/kokkos-core-wiki.git
git remote add upstream git@github.com:kokkos/kokkos-core-wiki.git
```
Of course, there is also an `--origin <name>` option you specify while cloning to give the remote created a different name than `origin` (untested):
```
git clone --origin ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git
git remote add origin git@github.com:kokkos/kokkos-core-wiki.git
```